src/librustc_data_structures/graph/scc/mod.rs

```diff
@@ -54,6 +54,11 @@ impl<N: Idx, S: Idx> Sccs<N, S> {
         self.scc_data.len()
     }
 
+    /// Returns the number of SCCs in the graph.
```

Comment:
(You should either delete this comment or revise it to something like "returns an iterator over the SCCs in the graph. Right now it just distracts, IMO...)

Reply:
((or even just "iterate over the SCCs in the graph", I guess...))